tests/inference/test_modeling_decoder.py

```diff
@@ -35,12 +35,13 @@
 DECODER_MODEL_ARCHITECTURES = ["gpt2"]
 
 
-@pytest.fixture(scope="module", params=[EXPORT_MODELS_TINY[model_arch] for model_arch in DECODER_MODEL_ARCHITECTURES])
+@pytest.fixture(scope="module", params=[MODEL_NAMES[model_arch] for model_arch in DECODER_MODEL_ARCHITECTURES])
 def export_model_id(request):
     return request.param
 
 
 @pytest.fixture(scope="module")
+@requires_neuronx
```

Comment:
You put the module fixture on the function, it will be executed before test functions even if they will be then skipped for not meeting the `requires_neuronx`.
Here is the error log, before adding `requires_neuronx` decorator, it was run for INF1 CI: https://github.com/huggingface/optimum-neuron/actions/runs/5869467161/job/15914595357#step:5:29